Makefile

```diff
@@ -0,0 +1,77 @@
+SHELL := /bin/bash
+
+ifndef LIGO
+LIGO=docker run --rm -v "$(PWD)":"$(PWD)" -w "$(PWD)" ligolang/ligo:0.44.0
```

Comment:
There is now a stable tag `ligolang/ligo:stable`
This would be targeted as LIGO version.
Since template are running as e2e test for each release of ligo it is safe for templates

Reply:
done, but not sure which version it is

Reply:
Sorry don't compile at this moment with stable cause `Variable "get_now" not found.`
It is 0.42.0 the rule is "stable is 2 versions behind the last version"

So it would remain 0.44.0 now and we will edit when stable will be 0.44